src/schema/root_query_type.ts

```diff
@@ -460,6 +464,33 @@ const RootQuery = new GraphQLObjectType({
         return reply;
       },
     },
+    koji_build_history_by_nvr: {
+      type: KojiHistoryType,
+      description:
+        'Retrieve history of tagging of a Koji build given its NVR or NVSC',
```

Comment:
Technically speaking : `or NSVC` is incorret. Koji build is associated with: NVR.
NVR has one format. NSVC has another format.

Reply:
Replace  `NVSC` with `NSVC`